vlsv_reader_parallel.cpp

```diff
@@ -434,9 +439,13 @@ namespace vlsv {
       // Only master process reads file footer and endianness. This is done using 
       // VLSVReader open() member function:
       if (myRank == masterRank) {
-         if (Reader::open(fname) == false) success = false;
+         if (Reader::open(fname) == false)
+         {
+             success = false;
+             cerr << "MASTER failed to open VLSV file, error " << Reader::getErrorString() << endl;
+         }
       }
-      if (success == false) cerr << "MASTER failed to open VLSV file" << endl;
+      if (success == false) 
```

Comment:
Is this if really supposed to not have braces and affect the MPI_Bcast operation below?

Reply:
Oh that is a derp.